recipe/build.sh

```diff
@@ -5,22 +5,22 @@ cp $BUILD_PREFIX/share/libtool/build-aux/config.* .
 # need macosx-version-min flags set in cflags and not cppflags
 export CFLAGS="$CFLAGS $CPPFLAGS"
 
+if [[ "${PKG_NAME}" == "libcurl-securetransport-static" ]]; then
+    USESSL="--with-secure-transport"
+    STATIC_BUILD="--enable-static --disable-shared"
+else
+    USESSL="--with-openssl=${PREFIX}"
+fi
 ./configure \
     --prefix=${PREFIX} \
     --host=${HOST} \
     --disable-ldap \
     --with-ca-bundle=${PREFIX}/ssl/cacert.pem \
-    --with-ssl=${PREFIX} \
+    $USESSL $STATIC_BUILD \
     --with-zlib=${PREFIX} \
     --with-gssapi=${PREFIX} \
     --with-libssh2=${PREFIX} \
     --with-nghttp2=${PREFIX} \
 || cat config.log
 
 make -j${CPU_COUNT} ${VERBOSE_AT}
-# TODO :: test 1119... exit FAILED
-# make test
```

Comment:
Keep this

Reply:
this was already commented out but we can try again :)

Reply:
ah you mean, keep as a comment, sure.

Reply:
thanks for reviewing!